### ISSUE TITLE
pg.v3 production ready?

### ISSUE THREAD

**Issue by owentran** [go-pg/pg#48]
Thanks for all your work with pg.v3. Any ideas of when you think it will be production-ready? I'll switch over my pg.v2 to v3 and file issues if you're ready for some user testing.

**Comment by vmihailenco** [comment]
It was marked "not ready for production", because I wanted to reserve right to make backwards incompatible changes (though I don't have any plans). But I use it in production and did not find any issues. So feel free to switch to the v3 and report all issues you find.

I will try to release v3 in 2-3 weeks.

**Comment by owentran** [comment]
Seeing an issue reading structs with pointers to time.Time...

```
panic: pg: unsupported src type: <time.Time Value> [recovered]
	panic: pg: unsupported src type: <time.Time Value>
```

**Comment by vmihailenco** [comment]
Please provide backtrace as well.

**Comment by vmihailenco** [comment]
@owentran I released v3.1.1 with a fix. Please try again.

**Comment by owentran** [comment]
Works. Thanks for the quick fix. I will continue to test on v3.

FYI, here was the backtrace

```
goroutine 30 [running]:
testing.func·006()
	/usr/local/go/src/testing/testing.go:441 +0x181
gopkg.in/pg%2ev3.appendValue(0xc2080d0000, 0x145, 0x2000, 0x6e53c0, 0xc20916d820, 0xd9, 0x0, 0x0, 0x0)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/encode.go:127 +0x460
gopkg.in/pg%2ev3.appendValue(0xc2080d0000, 0x145, 0x2000, 0x6e63e0, 0xc20916d820, 0x16, 0x0, 0x0, 0x0)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/encode.go:121 +0x272
gopkg.in/pg%2ev3.appendIface(0xc2080d0000, 0x145, 0x2000, 0x6e63e0, 0xc20916d820, 0x0, 0x0, 0x0)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/encode.go:111 +0x835
gopkg.in/pg%2ev3.(*pgValue).AppendValue(0xc208107900, 0xc2080d0000, 0x145, 0x2000, 0x6dcb20, 0xc20914e780, 0xd9, 0x0, 0x0, 0x0)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/typeinfo.go:88 +0x30b
gopkg.in/pg%2ev3.formatQuery(0xc2080d0000, 0x145, 0x2000, 0xc208093500, 0x2e7, 0x300, 0xc209185c78, 0x0, 0x1, 0x0, ...)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/encode.go:410 +0x968
gopkg.in/pg%2ev3.AppendQ(0xc2080d0000, 0x5, 0x2000, 0x8a6030, 0x2e7, 0xc209185c78, 0x1, 0x1, 0x0, 0x0, ...)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/encode.go:23 +0xe7
gopkg.in/pg%2ev3.writeQueryMsg(0xc20808b6c0, 0x8a6030, 0x2e7, 0xc209185c78, 0x1, 0x1, 0x0, 0x0)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/messages.go:137 +0xb8
gopkg.in/pg%2ev3.simpleQuery(0xc208030540, 0x8a6030, 0x2e7, 0xc209185c78, 0x1, 0x1, 0x2, 0x0, 0x0)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/db.go:372 +0x7f
gopkg.in/pg%2ev3.(*DB).Exec(0xc2080fce90, 0x8a6030, 0x2e7, 0xc209185c78, 0x1, 0x1, 0x0, 0x0, 0x0)
	/Users/owen/projects/go/src/gopkg.in/pg.v3/db.go:187 +0xe5
gopkg.in/pg%2ev3.(*DB).ExecOne(0xc2080fce90, 0x8a6030, 0x2e7, 0xc209185c78, 0x1, 0x1, 0x0, 0x0, 0x0)
```
